README.md

```diff
@@ -8,44 +8,113 @@ It also contains CLI interfaces for working with Lorentz contracts:
 - `lorentz-contract-param`: generate Michelson contract parameters
 - `lorentz-contract-storage`: generate Michelson initial storage values
 
-For more information, see the [quick start guide](https://assets.tqtezos.com/quickstart)
-or specifically the [client setup guide](https://assets.tqtezos.com/quickstart/1-index).
+## Installation
 
-Note: As of `09/27/2019` this package will not compile with the latest
-`stack`: `2.1.3`.  To build, you'll likely need
-[`stack v1.9.3`](https://github.com/commercialhaskell/stack/releases/tag/v1.9.3).
+### Use Pre-Built Binaries
+
+*Note:* Check https://github.com/tqtezos/lorentz-contract-param/releases/ for
+latest release versions
+
+### Linux (64-bit)
+
+``` bash
+wget https://github.com/tqtezos/lorentz-contract-param/releases/download/v1.2.2.1.0-1/lorentz-contract-param-1.2.2.1.0-1-linux-x86_64-static.tar.gz
```

Comment:
Can the `1.2.2.1.0-1` string be a bash variable so this readme doesn't need to be updated for new tags?